libraries/AP_Math/AP_Math.cpp

```diff
@@ -273,3 +273,18 @@ bool rotation_equal(enum Rotation r1, enum Rotation r2)
     return (v1 - v2).length() < 0.001;
 }
 
+// return horizontal distance between two positions in cm
```

Comment:
Should this really be moved?

There may still be room for a location.h

Reply:
I prefer not to confuse everybody by having to location file ...